examples/datadog/agent_otlp.yaml

```diff
@@ -0,0 +1,24 @@
+## Datadog Agent with OTLP enabled
+
+datadog:
+  apiKey: <DATADOG_API_KEY>
+  appKey: <DATADOG_APP_KEY>
+  clusterName: <CLUSTER_NAME>
+  tags: []
+  apm:
+    # Enable APM over Socket (Unix Socket or windows named pipe)
+    socketEnabled: true
+    socketPath: /var/run/datadog/apm.socket
+    hostSocketPath: /var/run/datadog/
```

Comment:
Nit: APM isn't technically necessary for OTLP. As that data will actually go through the `agent` instead of the `trace-agent`. I guess generally there likely are a lot of customers using both though so this isn't going to harm anything but isn't needed.

Reply:
To keep it minimal, I would remove it.